### ISSUE TITLE
Virus in zip file

### ISSUE THREAD

**Issue by keithxemi** [QQuick/Transcrypt#266]
Microsoft Security Essentials reports Trojan: Win32/Spursint.F!cl in the file Transcrypt-master.zip and Transcrypt-3.6.4.zip https://www.microsoft.com/security/portal/threat/encyclopedia/entry.aspx?name=Trojan%3aWin32%2fSpursint.F!cl&threatid=2147717281&enterprise=0

**Comment by JdeH** [comment]
Thanks!

Will look into this a.s.a.p..

Kind regards
Jacques

**Comment by JdeH** [comment]
Hi Keith

First I've scanned both files with ESET NOD32.

After that I've installed Windows 10 with the newest version of Windows Defender.
I scanned both files again with Windows Defender, after downloading them from GitHub and PyPi.

In addition to this I scanned my whole Transcrypt working directory with Windows Defender.

Unfortunately (or fortunately) I could not reproduce your findings.
Might there be a local problem on your PC?
Or did you perhaps obtain these files directly from a 3rd party rather then from PyPi or GitHub?
Please keep me informed on this.

Thanks for the report anyhow, better safe then sorry!
Jacques

**Comment by keithxemi** [comment]
Perhaps this is unique to Windows 7? I have no problem with other zip downloads. Here is the detail from MSE with the specific file name in both zips:
Items: 
containerfile:C:\Users\Keith\Downloads\Transcrypt-3.6.4.zip
file:C:\Users\Keith\Downloads\Transcrypt-3.6.4.zip->Transcrypt-3.6.4/transcrypt/demos/cyclejs_demo/component_demos/labeled_slider/labeled_slider_js_orig.js
webfile:C:\Users\Keith\Downloads\Transcrypt-3.6.4.zip|https://pypi.python.org/packages/51/c4/54cd5c2fc4391e612e9ffc6850bed03be38b0395f22c35cad719a1a75b1e/Transcrypt-3.6.4.zip|chrome.exe
And this:
Items: 
containerfile:C:\Users\Keith\Downloads\Transcrypt-master.zip
file:C:\Users\Keith\Downloads\Transcrypt-master.zip->Transcrypt-master/transcrypt/demos/cyclejs_demo/component_demos/labeled_slider/labeled_slider_js_orig.js
webfile:C:\Users\Keith\Downloads\Transcrypt-master.zip|https://codeload.github.com/QQuick/Transcrypt/zip/master|chrome.exe

**Comment by JdeH** [comment]
I've scanned the file labeled_slider_js_orig.js separately bu I am unable to reproduce the problem.
Since I don't have Windows 7 at my disposal but your report is quite clear and I am curious what's going on. Anyone else experiencing this problem on Windows 7?

@keithxemi 
Could you please mail one of the infected files to jdeh@transcrypt.org, allowing me to investigate further.

**Comment by keithxemi** [comment]
I have submitted my downloaded zip to https://www.virustotal.com/ and  https://virusscan.jotti.org/ with the same results, nothing found. I cloned this repository and scanned with Microsoft SE and found no virus. So I can use the clone files with no worry and I believe this is probably a false positive. As you say the flagged file is just plain js. However, that gmail also refused to attach the zip says there is cause for concern.
